### PR TITLE
chore: async NFTScan support chain config

### PR DESCRIPTION
**Pull request by Lanttcat**
## Description

Closes # (NO_ISSUE)

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
  - [ ] I have removed all in development `console.log`s
  - [ ] I have removed all commented code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

**Comment by Lanttcat** [comment]
Please hold this pr before the worker repo deploy:  https://github.com/DimensionDev/Cloudflare-workers/pull/56

File: packages/web3-providers/src/NFTScan/helpers/EVM.ts [hunk 1]
```diff
@@ -32,15 +32,24 @@ import { resolveActivityType } from '../../helpers/resolveActivityType.js'
...
+    | ChainId.Avalanche
 
 export const resolveHostName = createLookupTableResolver<NFTScanChainId, string>(
     {
-        [ChainId.Mainnet]: 'https://www.nftscan.com',
+        [ChainId.Mainnet]: 'https://restapi.nftscan.com',
         [ChainId.Matic]: 'https://polygon.nftscan.com',
         [ChainId.BSC]: 'https://bnb.nftscan.com',
         [ChainId.Arbitrum]: 'https://arbitrum.nftscan.com/',
+        [ChainId.Avalanche]: 'https://avaxapi.nftscan.com/',
```
Comment (guanbinrui) [inline packages/web3-providers/src/NFTScan/helpers/EVM.ts]:
```suggestion
        [ChainId.Avalanche]: 'https://avax.nftscan.com/',
```

File: packages/web3-providers/src/NFTScan/helpers/EVM.ts [hunk 2]
```diff
@@ -32,15 +32,24 @@ import { resolveActivityType } from '../../helpers/resolveActivityType.js'
...
         [ChainId.Optimism]: 'https://optimism.nftscan.com/',
+        [ChainId.xDai]: 'https://cronosapi.nftscan.com/',
```
Comment (guanbinrui) [inline packages/web3-providers/src/NFTScan/helpers/EVM.ts]:
```suggestion
        [ChainId.xDai]: 'https://cronos.nftscan.com/',
```

File: packages/web3-providers/src/NFTScan/helpers/EVM.ts [hunk 3]
```diff
@@ -32,15 +32,24 @@ import { resolveActivityType } from '../../helpers/resolveActivityType.js'
...
+    | ChainId.BSC
+    | ChainId.Arbitrum
+    | ChainId.Optimism
+    | ChainId.xDai
+    | ChainId.Avalanche
 
 export const resolveHostName = createLookupTableResolver<NFTScanChainId, string>(
     {
-        [ChainId.Mainnet]: 'https://www.nftscan.com',
+        [ChainId.Mainnet]: 'https://restapi.nftscan.com',
```
Comment (guanbinrui) [inline packages/web3-providers/src/NFTScan/helpers/EVM.ts]:
```suggestion
        [ChainId.Mainnet]: 'https://www.nftscan.com',
```

**Review by guanbinrui — APPROVED** [review]

**Pull request merged**